lib/src/components/NxTag/__tests__/NxTag.test.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+import { faPlusCircle, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
+
+import NxTag, { NxSelectableTag } from '../NxTag';
+import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+describe('NxTag', function() {
+  const getShallowComponent = enzymeUtils.getShallowComponent(NxTag, { children: 'basic tag' });
+
+  it('renders NxTag with the `nx-tag` class', function() {
+    expect(getShallowComponent().find('.nx-tag')).toExist();
+  });
+
+  it('sets the nx-tag--default class if no color prop is passed', function() {
+    expect(getShallowComponent().find('.nx-tag--default')).toExist();
+  });
+
+  it('sets the color class using the color if it is provided', function() {
+    const colorComponent = getShallowComponent({ color: 'orange' });
+    expect(colorComponent.find('.nx-tag--orange')).toExist();
+    expect(colorComponent.find('.nx-tag--default')).not.toExist();
+  });
+});
```

Comment:
Either we should add a test here that checks for the `NxOverflowTooltip`, or we should be sure to capture the text overflow in a visual test.

Reply:
> Either we should add a test here that checks for the `NxOverflowTooltip`, or we should be sure to capture the text overflow in a visual test.

Visual test: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/c575a565138970f94db380581dfd5aa4529c23d3